perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -472,6 +474,22 @@ sub inspect : Private {
                             $c->cobrand->max_detailed_info_length
                         );
                 }
+            } elsif ( $c->get_param('detailed_information') eq '' ) {
```

Comment:
I see this bug fix, wonder if should be a separate commit just fixing that? No biggie.